packages/edit-post/src/components/secondary-sidebar/list-view-sidebar.js

```diff
@@ -152,7 +179,11 @@ export default function ListViewSidebar() {
 				className="edit-post-editor__list-view-container"
 			>
 				{ tab === 'list-view' && (
-					<div className="edit-post-editor__list-view-panel-content">
+					// eslint-disable-next-line jsx-a11y/no-static-element-interactions
```

Comment:
I'm torn between asking for a "disable reason" comment here (in general it's good to add one when we disable a lint rule) and wondering if we should just get rid of this rule altogether, given that we're disabling it in so many places already 😅